src/pystack/_pystack/pythread.cpp

```diff
@@ -96,9 +96,20 @@ findPthreadTidOffset(
         auto pthread_id_addr = manager->getField(current_thread, &py_thread_v::o_thread_id);
 
         // Attempt to locate a field in the pthread struct that's equal to the pid.
-        uintptr_t buffer[200];
-        manager->copyObjectFromProcess(pthread_id_addr, &buffer);
-        for (int i = 0; i < 200; i++) {
+        uintptr_t buffer[100];
+        size_t buffer_size = sizeof(uintptr_t) * 100;
```

Comment:
it's a nit, but I strongly prefer `size_t buffer_size = sizeof(buffer);` - it removes duplication (`100` twice, `uintptr_t` twice) and thus eliminates the chances of a mistake that a compiler won't catch if we adjust one of those.
```suggestion
        size_t buffer_size = sizeof(buffer);
```